tools/storyteller/StoryWriter.java

```diff
@@ -91,13 +84,36 @@ public void saveScreenshot(String project, String story) {
       String filenameWithoutExtension =
           Paths.get(getUnsharedStoriesPath(), getCurrentTimeString()).toString();
       ImageIO.write(screenshot, "jpg", Paths.get(filenameWithoutExtension + ".jpg").toFile());
-      // Replace empty with one space, so that the prototxt will have the entry
-      story = story.isEmpty() ? " " : story;
-      fileUtils.writePrototxt(
-          ScreenshotMetadata.newBuilder().setProject(project).setOneliner(story).build(),
-          filenameWithoutExtension + ".prototxt");
     } catch (AWTException | IOException e) {
       log.error("Error in saving screenshot", e);
     }
   }
+
+  private void updateCurrentStory(String project) {
+    currentStory = Story.newBuilder()
```

Comment:
You can change currentStory to `Story.Builder` so you don't have to build every time.
Just so you know, you can turn any proto object into a builder with the current data.
So, `currentStory` is equal to `currentStory.toBuilder().build()`